### PR TITLE
Python: Remove need to run from src

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

**Comment by mwestphal** [comment]
@snoyer @Meakk please review

**Comment by snoyer** [comment]
Could run Black to add empty lines around the functions in `conftest.py` but otherwise looks good

File: python/testing/test_image_compare.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+
 import f3d
 
 
 def test_compare_with_file():
-    dataset = "./testing/data/cow.vtp"
-    reference = "./testing/baselines/TestPythonCompareWithFile.png"
+    testingDir = str(Path(__file__).parent.parent.parent / "testing")
+    dataset = testingDir + "/data/cow.vtp"
+    reference = testingDir + "/baselines/TestPythonCompareWithFile.png"
```
Comment (snoyer) [inline python/testing/test_image_compare.py:11]: should be fully pythonized now ;p
```suggestion
    testing_dir = Path(__file__).parent.parent.parent / "testing"
    dataset = f"{testing_dir}/data/cow.vtp"
    reference = f"{testing_dir}/baselines/TestPythonCompareWithFile.png"
```

File: python/testing/test_image_compare.py [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 import tempfile
+
 import f3d
 
 
 def test_compare_with_file():
-    dataset = "./testing/data/cow.vtp"
-    reference = "./testing/baselines/TestPythonCompareWithFile.png"
+    testing_dir = Path(__file__).parent.parent.parent / "testing"
+    dataset = f"{testing_dir}/data/cow.vtp"
```
Comment (Meakk) [inline python/testing/test_image_compare.py:10]: This string formatting is better, but let's do it for the other test too.

**Review by Meakk — CHANGES_REQUESTED** [review]

**Review by Meakk — APPROVED** [review]

**Pull request merged**